shorthand/FhirDataEvaluatorIG/input/resources/StructureDefinition-cqfm-aggregateMethod.json

```diff
@@ -0,0 +1,244 @@
+{

```

Comment:
This should not go into our repo. Please try to reference it via a FHIR package.